sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -364,7 +364,7 @@ object SQLConf {
         "to insert a bloom filter in the other side to reduce the amount of shuffle data.")
       .version("3.3.0")
       .booleanConf
-      .createWithDefault(false)
+      .createWithDefault(true)
```

Comment:
Hmm, is this change for master or 3.3? Seems this is a new feature added in 3.3, isn't it safer to keep it as false by default? Usually we choose to be conservative instead of aggressive on enabling new feature.

Reply:
It's for master only, and will not be merged to 3.3. 
This way, new PRs to the master will less likely to break this feature.

Reply:
I see. makes sense.

Reply:
@sunchao Maybe you can enable vectorized Parquet reader by default in master branch only too. So it won't be silently broken.

Reply:
Thanks, that's a good idea. Let me switch it.